docs/Build.md

```diff
@@ -28,8 +28,10 @@ See the project's `README` for detailed information about installing `hie`.
 The build script `install.hs` defines several targets using the `shake` build system. The targets are roughly:
 
 * `hie-*`: builds and installs the `hie` binaries. Also renames the binaries to contain the correct version-number.
-* `build`: builds and installs `hie` binaries for all supported `ghc` versions.
+* `build-lastest`: builds ad installs `hie` for the last available and supported `ghc` version.
```

Comment:
Typo `latest`

Reply:
typo `ad`

Reply:
s/last/latest/ maybe?

Reply:
lastest could be correct too, no? (to use the same name that the target)

Reply:
lastest is wrong, imo. Either `last` or `latest`.

Reply:
Ok i will change all references to lastest to latest (including the target name)

Reply:
done!